### ISSUE TITLE
Experiment disabled but no option to enable it

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot#1810]
Ref https://github.com/mozilla/activity-stream/issues/1749,

> When I click on the Test Pilot button, it shows no green checkmark beside Activity Stream. However when I go to the Activity Stream page, it gives me a button to disable it and no button to enable it, implying that it's already active. However, it doesn't appear to be active, as I don't see any of its UI on the new-tab page.
> Disabling then re-enabling it fixed the problem, but I figured I'd report it anyway.
> ...
> At the moment it looks fine (as yours), but as I say I fixed the problem by disabling & re-enabling it. So I don't know if there's anything I can really investigate, unfortunately. Although if it helps, I *think* I originally disabled it via the "This addon is slowing down Firefox" warning at the bottom.

**TL;DR:** The theory is that the original reporter disabled the add-on via the "This Addon is slowing down Firefox" style doorhanger. But then the Test Pilot site still reports the add-on as installed. So it's in a weird state of disabled but installed. User had to disable and reenable the add-on manually to get it reinstalled correctly.

**Comment by SoftVision-PaulOiegas** [comment]
Probably because that doorhanger with "add-on slowing down" is manually disabling it and we are not detecting yet the manual disable of the add-ons from about:addons page. 
This is already tracked in #1300 bug so I think we can close this as a dupe.
